docs/database-views/input4MIPs_delivery-summary_CMIP6Plus.html

```diff
@@ -113,9 +113,9 @@ <h4>
     </tr>
     <tr>
       <td>10</td>
-      <td><b><a href='https://pcmdi.llnl.gov/mips/amip' target='_blank'>AMIP sea-surface temperature and sea-ice boundary forcing</a></b></br>Internal docs: <a href='../dataset-overviews/amip-sst-sea-ice-boundary-forcing' target='_blank'>here</a></td>
+      <td><b><a href='https://github.com/PCMDI/amipbcs' target='_blank'>AMIP sea-surface temperature and sea-ice boundary forcing</a></b></br>Internal docs: <a href='../dataset-overviews/amip-sst-sea-ice-boundary-forcing' target='_blank'>here</a></td>
```

Comment:
I wouldn't bother to change this - site should be up ~soon

Reply:
Reverted